apps/govern/components/Contracts/Votes.tsx

```diff
@@ -0,0 +1,155 @@
+const columns: ColumnsType<MyVote> = [
+  {
+    title: 'Staking contract',
+    key: 'name',
+    render: (_, record) =>
+      record.address && record.chainId ? (
+        <Space size={2} direction="vertical">
+          <Link href={`/contracts/${record.address}`}>{record.name}</Link>
+          <Text type="secondary">{CHAIN_NAMES[record.chainId] || record.chainId}</Text>
+        </Space>
+      ) : (
+        <Tooltip
+          color={COLOR.WHITE}
+          overlayStyle={{ maxWidth: '500px' }}
+          title={
+            <Paragraph className="m-0">
+              Unused staking incentives are retained unminted in the Rollover Pool. Unused staking
+              incentives are retained unminted in the Rollover Pool. They can be made available in a
+              future epoch by calling the <b>retain</b> method.
+            </Paragraph>
+          }
+        >
+          <Text type="secondary">
+            {record.name} <InfoCircleOutlined className="ml-8" />
+          </Text>
+        </Tooltip>
+      ),
+    width: 200,
+  },
+  {
+    title: 'Current weight',
+    key: 'weight',
+    render: (_, record) => <Text>{`${record.currentWeight}%`}</Text>,
+    width: 120,
+  },
+  {
+    title: 'Next weight',
+    key: 'nextWeight',
+    render: (_, record) => <Text>{`${record.nextWeight}%`}</Text>,
+    width: 140,
+  },
+];
+
+const rowClassName = (record: MyVote): string => {
+  return record.isRetainer ? 'highlight-row' : '';
+};
+
+export const Votes = () => {
+  const { stakingContracts } = useAppSelector((state) => state.govern);
+  const { lastUserVote, userVotes } = useAppSelector((state) => state.govern);
+
+  const votesBlocked = useMemo(
+    // TODO: unblock somehow, once counter is finished
+    () => (lastUserVote !== null ? lastUserVote + TEN_DAYS_IN_MS > Date.now() : false),
+    [lastUserVote],
+  );
+
+  const data: MyVote[] = useMemo(() => {
+    const userVotesArray = Object.entries(userVotes);
+    if (userVotesArray.length > 0 && stakingContracts.length > 0) {
+      return userVotesArray.reduce((res: MyVote[], [key, value]) => {
+        const contract = stakingContracts.find((item) => item.address === key);
+        if (contract) {
+          res.push({
+            address: contract.address,
+            name: contract.metadata?.name,
+            chainId: contract.chainId,
+            currentWeight: value.current.power,
+            nextWeight: value.next.power,
+          });
+        }
+
+        if (key === getBytes32FromAddress(RETAINER_ADDRESS)) {
+          res.push({
+            name: 'Rollover pool',
+            currentWeight: value.current.power,
+            nextWeight: value.next.power,
+            isRetainer: true,
+          });
+        }
+        return res;
+      }, []);
+    } else {
+      return [];
+    }
+  }, [userVotes, stakingContracts]);
+
+  return (
+    <VotesRoot>
+      {votesBlocked && (
+        <Flex gap={16} align="center" justify="end">
+          <Text type="secondary">
```

Comment:
please wrap the <Text> block in the lastUserVote condition too

Reply:
good point!